tests/smoke/apps_test.go

```diff
@@ -21,4 +26,26 @@ var _ = Describe("apps", func() {
 			HaveHTTPBody(ContainSubstring("Hi, I'm not Dora!")),
 		))
 	})
+
+	When("getting apps manifest", func() {
```

Comment:
I would put this context in its own file, something like `app_manifest.go`. The rest of the tests in this file are exercising the `cf app` command. `cf create-app-manifest` is a completely different command